### ISSUE TITLE
Update for 2016 analysis 

### ISSUE THREAD

**Issue by josibake** [Code4HR/okcandidate-analysis#2]
Create a folder and organize a place for 2016 analysis results. @wbprice I'm assuming there is only data for vbeach?

**Comment by wbprice** [comment]
That's right.  OKCandidate was only active in Va Beach for the November 2016 election.
